bitnami/jenkins/templates/ingress.yaml

```
@@ -45,9 +45,9 @@ spec:
             {{- end }}
             backend: {{- include "common.ingress.backend" (dict "serviceName" (include "common.names.fullname" $) "servicePort" "http" "context" $) | nindent 14 }}
     {{- end }}
-  {{- if or (and .Values.ingress.tls (or .Values.ingress.certManager .Values.ingress.selfSigned)) .Values.ingress.extraTls }}
+  {{- if or Values.ingress.tls Values.ingress.extraTls }}
```

`.Values` is missing the initial dot (`.`)

```suggestion
  {{- if or .Values.ingress.tls .Values.ingress.extraTls }}
```

---

Thank you @migruiz4, I fixed it and bumped up the version.